cmd/dfget/app/root.go

```diff
@@ -64,16 +64,16 @@ func init() {
 
 // runDfget does some init operations and starts to download.
 func runDfget() error {
+	// get config from property files
+	initProperties()
+
```

Comment:
you make init config before init log,I have see when init config  it  put out log by `Logrus.debugf`,so there may be something wrong.There are no place the log  put out. You'd  better make sure it